src/js/Node.js

```diff
@@ -1791,8 +1791,22 @@ export class Node {
           this.dom.value.parentNode.appendChild(this.dom.date)
         }
 
-        this.dom.date.innerHTML = new Date(value).toISOString()
-        this.dom.date.title = new Date(value).toString()
+        let title = null
```

Comment:
I try to use `const` instead of `let` as much as possible, it often simplifies the code. In this case it is possible to do something like:

```js
const title = (typeof this.editor.options.timestampFormat === 'function') 
  ? this.editor.options.timestampFormat({ ... })
  : new Date(value).toISOString()

// ... no need for the if (!title) {... } else { ... }
while (this.dom.date.firstChild) {
  this.dom.date.removeChild(this.dom.date.firstChild)
}
this.dom.date.appendChild(document.createTextNode(title))
```

That is less if/elses, and also clearly splits formatting logic and rendering logic.

Reply:
`if (!title) {... } else { ... }` is necessary. If `timestampFormat` return null, we need to fallback to the default setting.

Reply:
Ah, that's a good point. You could probably solve it like 

```js
const title = (hasTimeStampFormat && timeStampFormat()) || defaultTimeStampFormat()
```
but it's ok with me either way :+1:

Reply:
Yeah,  I have tried this solution, but I think that's too long and not so readable. Maybe that's because I'm a backend.:) Feel free to modify my code. You can modify it as your way.